### ISSUE TITLE
Forms Interactive Do's & Don'ts

### ISSUE THREAD

**Issue by dahliaferrer** [grommet/hpe-design-system#2699]
Related to 2648, I have finished the content writing, added guidance on error messaging and validations, however the Do's and Don'ts are a long, boring list that will turn off our users. Should create an interactive version of the Do's and Don'ts that shows them the guidance in action.

**Comment by dahliaferrer** [comment]
I've duplicated Beza's Figma file and edited it to get closer to what I think we can do to best display the do's and don'ts. 

I've sent it to developer @britt6612 who is out of office for a few days. When Brittany returns the question will be: is my Figma file in good enough shape for her to work off of, or should I send it back to Beza for her to clean up so all of the margins are perfect, spacing, etc.

https://www.figma.com/file/xXrcOwPzv04JrL54MYzywN/DS-Forms-guidance-DAHLIA?node-id=1%3A6

**Comment by bezAdane** [comment]
@dahliaferrer The design you made edits have already been reviewed by Scott and myself, have you confirmed with him to make those edits? We usually make comments on a design if you have questions, I'm confused to why you copied a design to a new file and change it. If possible please make your suggestions of copy write by using  "Add comments" on the original design of form component library and i can review to make the necessary changes. I want to make sure we all are aligned on the process. Link can be found [here](https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=6813%3A12781)  and [here](https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=6709%3A12594) where the designs that has been reviewed and finalized to Do's and Don't visual example. Once the templet you are working is done we can finalize it to where in the guidance it can be added. @SOjaHPE please comment, thank you.

**Comment by britt6612** [comment]
This can be broken up into 2 tickets this shouldn't wait until my availability. Any dev can pick this up once the designs are ready. So we can have this ticket be a design and then make a new ticket for the dev side of it. This is just out of the process and it seems like there is some confusion. @dahliaferrer I know you made the Figma file however there are no comments on there so did anyone approve this? We usually just comment on the original Figma that @bezAdane made.

**Comment by taysea** [comment]
Close, stale.
